include/mbedtls/config.h

```diff
@@ -2798,7 +2798,26 @@
 //#define MBEDTLS_SSL_CACHE_DEFAULT_MAX_ENTRIES      50 /**< Maximum entries in cache */
 
 /* SSL options */
-//#define MBEDTLS_SSL_MAX_CONTENT_LEN             16384 /**< Maxium fragment length in bytes, determines the size of each of the two internal I/O buffers */
+//#define MBEDTLS_SSL_MAX_CONTENT_LEN             16384 /**< Maximum fragment length in bytes, determines the size of each of the two internal I/O buffers */
+
+//#define MBEDTLS_SSL_IN_CONTENT_LEN              16384 /**< Optionally override MBEDTLS_SSL_MAX_CONTENT_LEN for the inward TLS buffer only. */
+
+/**
+ * It is possible to save RAM by setting a smaller outward buffer using
```

Comment:
This should be a doxygen comment. e.g. open with `\def MBEDTLS_SSL_OUT_CONTENT_LEN` or similar.